package.json

```diff
@@ -18,6 +18,7 @@
     "lucide-react": "^0.446.0",
     "next": "14.2.13",
     "package.json": "^2.0.1",
+    "prettier": "^3.3.3",
```

Comment:
Can you add this as part of scripts?

Reply:
Also if you can add this as a Github workflow would be helpful, may be as separate PR this can be taken up.

Reply:
@tarun-29 please feel free to create new issue and fix this.